functional.py

```diff
@@ -0,0 +1,83 @@
+# Below you will find several functions I attempted to invent
+
+## 1. sequential_map
+### This functions takes any amount of functions and a container as an input
+### It returns a list of values from initial container modified by consequent use of functions from input
+
+def sequential_map(*args):
+    def apply(func, container):
+        return [func(item) for item in container]
+    data = args[-1]
```

Comment:
Удобно сделать так, потом тогда не пришлось бы делать срезы
```suggestion
    *functions, data = args
```